AppCenter/AppCenter/Internals/HttpClient/MSHttpClient.m

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSHttpClient.h"
+
+@implementation MSHttpClient
+
+- (instancetype)init {
+
+  // TODO implement this properly.
+  return self = [super init];
+}
+
+- (instancetype)initWithMaxHttpConnectionsPerHost:(int)maxHttpConnectionsPerHost {
+  if ((self = [super init])) {
```

Comment:
Usually inits are cascading. That means you would call `[self init]` instead of `[super init]`